examples/tts/conf/rad-tts_feature_pred.yaml

```diff
@@ -0,0 +1,335 @@
+name: RadTTS_logs
```

Comment:
Can you name this file and the `name` parameter something appropriate?

Reply:
renamed it